MFVNeutralino/test/TrackMover/hists.cc

```diff
@@ -489,7 +542,7 @@ int main(int argc, char** argv) {
       const double tw = tau_weight(tau);
       h_tau->Fill(tau, tw);
       h_tau_tw->Fill(tau, tw, 1.0);
-      w *= tw;
+      //w *= tw; // Old procedure did reweight by 10mm to a desire ctau 
```

Comment:
Wait, don't we do this still? I thought we model the lifetimes in TM in some manner...